app/models/user_subject_queue.rb

```diff
@@ -22,17 +22,17 @@ def self.scope_for(action, groups, opts={})
 
   def self.enqueue_subject_for_user(user: nil, workflow: nil, set_member_subject: nil)
```

Comment:
Thought: I assume the front end will only really know about subjects here so when submitting a subject to the queue, it'll probably be similar to the classification submitting subject_ids.